coordinator/src/main/java/org/apache/uniffle/coordinator/metric/CoordinatorMetrics.java

```diff
@@ -96,6 +98,23 @@ public static void updateDynamicGaugeForRemoteStorage(String storageHost, double
     }
   }
 
+  public static void addDynamicGaugeForUser(String user) {
+    if (!StringUtils.isEmpty(user)) {
+      GAUGE_APP_NUM_TO_USER.computeIfAbsent(
+          user, x -> metricsManager.addGauge(APP_NUM_TO_USER + user));
+      GAUGE_APP_NUM_TO_USER.get(user).inc();
```

Comment:
Could we merge this line to the pre line?

Reply:
Do you mean to merge 105 lines into 104 ?

Reply:
Yes

Reply:
Maybe not. I must set this value after the add metric, but the `computeIfAbsent` method must return type `Guage`

Reply:
I mean GAUGE_APP_NUM_TO_USER.computeIfAbsent(
          user, x -> metricsManager.addGauge(APP_NUM_TO_USER + user)).inc()

Reply:
Got it.